src/lm_buddy/jobs/_entrypoints/prometheus.py

```diff
@@ -0,0 +1,165 @@
+# lm-buddy entrypoint to run evaluations using a Prometheus inference server
+# see https://github.com/kaistAI/prometheus/blob/main/evaluation/benchmark/run_absolute_scoring.py
+
+import copy
+import json
+from pathlib import Path
+
+from datasets import Dataset, load_dataset
+from fastchat.conversation import get_conv_template
+from openai import Completion, OpenAI, OpenAIError
+from tqdm import tqdm
+from transformers import PreTrainedTokenizer
+
+from lm_buddy.integrations.huggingface import HuggingFaceAssetLoader
+from lm_buddy.integrations.huggingface.tokenizer_config import AutoTokenizerConfig
+from lm_buddy.integrations.wandb import (
+    ArtifactLoader,
+    ArtifactType,
+    build_directory_artifact,
+    wandb_init_from_config,
+)
+from lm_buddy.jobs.common import LMBuddyJobType
+from lm_buddy.jobs.configs import PrometheusJobConfig
+
+
+class BadResponseError(Exception):
```

Comment:
since this class doesn't have any methods but only handles variables, it could be a dataclass instead:

```python
from dataclasses import dataclass

@dataclass
class BadResponseError(Exception):
    message: str
    error: str = None
```